src/graphql/mutations/createEtchPacket.js

```diff
@@ -2,7 +2,14 @@
 const defaultResponseQuery = `{
   eid
   name
+  status
+  isTest
+  isFree
+  containsFillData
+  payload
+  numberRemainingSigners
   detailsURL
+  webhookURL
```

Comment:
These are all the things that are possible. Do we want them all?

Reply:
Keep these new ones:

`status, isTest, numberRemainingSigners, webhookURL`

Reply:
Maybe add the date stamps like the query response too?